src/SingleProject/Resizetizer/src/ResizetizerPackages.projitems

```diff
@@ -18,6 +18,8 @@
     <PackageReference Include="ShimSkiaSharp" GeneratePathProperty="true" PrivateAssets="all" />
     <PackageReference Include="Fizzler" GeneratePathProperty="true" PrivateAssets="all" />
     <PackageReference Include="ExCSS" GeneratePathProperty="true" PrivateAssets="all" />
+    <PackageReference Include="System.Drawing.Common" GeneratePathProperty="true" PrivateAssets="all" />
```

Comment:
Do we know why this is here? What is pulling it in or is it the netstandard TFM? Looking at the project.assets.josn all the System.Drawing is a dll in the ref of NETStandard.Library/2.0.3.